src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -1728,8 +1728,10 @@ void java_lang_Thread::set_jvmti_thread_state(oop java_thread, JvmtiThreadState*
   java_thread->address_field_put(_jvmti_thread_state_offset, (address)state);
 }
 
-void java_lang_Thread::clear_extentLocalBindings(oop java_thread) {
-  java_thread->obj_field_put(_extentLocalBindings_offset, NULL);
+void java_lang_Thread::clear_scopedValueBindings(oop java_thread) {
```

Comment:
It looks like there is only one caller of this method that takes an oop.  Would it make sense to have the caller check for the null oop, and assert that the oop is not null here?

Reply:
Done.